code/android-core-library/src/androidTest/java/com/adobe/marketing/mobile/AndroidV4ToV5MigrationTests.java

```diff
@@ -525,6 +535,8 @@ public void testDataMigration_OnlyPrivacyKeyInV4_onlyMigratesConfiguration() thr
         // Target data should not get migrated or removed
         assertTrue(v4DataStore.contains(V4.Target.TNT_ID));
         assertTrue(v4DataStore.contains(V4.Target.THIRD_PARTY_ID));
+        assertTrue(v4DataStore.contains(V4.Target.SESSION_ID));
+        assertTrue(v4DataStore.contains(V4.Target.THIRD_PARTY_ID));
```

Comment:
Fix assertion here for Edge host 
```
assertTrue(v4DataStore.contains(V4.Target.EDGE_HOST));
```

Reply:
Assert v5 Target data store doesn't contain session ID and edge host too when privacy is opted out L561-562

Reply:
I've pushed the feedback updates to this pr @praveek